packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -965,7 +966,17 @@ export default function ToolpadApp({
           <AppThemeProvider dom={dom}>
             <CssBaseline enableColorScheme />
             {version === 'preview' && !hidePreviewBanner ? (
-              <Alert severity="info">This is a preview version of the application.</Alert>
+              <Alert severity="info">
+                This is a preview version of the application.
+                <Button
+                  variant="contained"
+                  size="small"
+                  href={`/_toolpad/app/${appId}`}
+                  sx={{ m: '-14px 16px -8px', fontSize: '10px' }}
```

Comment:
Should we not make `px` explicit here and just use a number for the values? 
Seems like that's the pattern we usually follow.

Also does it really need all these negative/different margins? Just because this layout seems kind of simple at first glance.

Reply:
Perhaps use row flexbox on the parent with `align-items: center` and `gap`?